### PR TITLE
AF-1079 : Empty "Validation errors" popup in Decision/Business Central

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/AF-1079

https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/725
https://github.com/kiegroup/appformer/pull/259
https://github.com/kiegroup/kie-wb-common/pull/1537
https://github.com/kiegroup/drools-wb/pull/835

### CODE REVIEW

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by barboras7** [comment]
Jenkins please retest this

**Comment by Rikkola** [comment]
These randomly failing unit tests are killing the rainforests.
Jenkins please retest this.

**Comment by Rikkola** [comment]
@barboras7 @karreiro I added additional protection against concurrent validations from multiple users. The change is in a separate commit.

**Comment by Rikkola** [comment]
jenkins please retests this

File: kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/validation/asset/ValidatorBuildService.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ public ValidatorBuildService(final @Named("ioStrategy") IOService ioService,
         this.buildInfoService = buildInfoService;
     }
 
-    public List<ValidationMessage> validate(final Path resourcePath,
+    public synchronized List<ValidationMessage> validate(final Path resourcePath,
```
Comment (karreiro) [inline kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/validation/asset/ValidatorBuildService.java]: I think that there's a code style issue here.

**Comment by barboras7** [comment]
jenkins please retests this

**Comment by Rikkola** [comment]
Might have better luck playing to lotto than doing this, but here goes.

jenkins please retests this

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
@Rikkola Should we apply [this](https://github.com/kiegroup/kie-wb-common/pull/1537/commits/060b7e9e02e3abdf9602749179ccba4b7de0a9ba) in the 7.5.x branch too?

**Comment by Rikkola** [comment]
@karreiro Well spotted. Missed one branch.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by ederign** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by Rikkola** [comment]
Tempted to think something is actually broken here. Trying it out locally.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by barboras7** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by Rikkola** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
@ederign by merging just one PR of ensemble we introduced incompatible changes for other master PRs, could we please merge ASAP whole AF-1079 ensemble?

**Comment by Rikkola** [comment]
@jomarko Just trying to get the builds green.

**Comment by Rikkola** [comment]
jenkins please retest this

**Comment by manstis** [comment]
@Rikkola BTW did you realise this PR also contained a commit for "DROOLS-2446 : Concurrent spreadsheet table validation...".. Since the PR was approved I merged keeping separate commits... but you might want to check this is what you intended :-)

**Pull request merged**